frontend/src/metabase/visualizations/components/settings/ChartSettingsTableFormatting.jsx

```diff
@@ -9,24 +9,39 @@ import Radio from "metabase/components/Radio";
 import Toggle from "metabase/components/Toggle";
 import ColorPicker from "metabase/components/ColorPicker";
 import PopoverWithTrigger from "metabase/components/PopoverWithTrigger";
+import NumericInput from "metabase/components/NumericInput";
 
 import { SortableContainer, SortableElement } from "react-sortable-hoc";
 
 import MetabaseAnalytics from "metabase/lib/analytics";
-import { formatNumber, capitalize } from "metabase/lib/formatting";
-import { isNumeric } from "metabase/lib/schema_metadata";
+import { capitalize } from "metabase/lib/formatting";
+import { isNumeric, isString } from "metabase/lib/schema_metadata";
 
 import _ from "underscore";
 import d3 from "d3";
 import cx from "classnames";
 
-const OPERATOR_NAMES = {
+const NUMBER_OPERATOR_NAMES = {
   "<": t`less than`,
   ">": t`greater than`,
   "<=": t`less than or equal to`,
   ">=": t`greater than or equal to`,
   "=": t`equal to`,
   "!=": t`not equal to`,
+  "is-null": t`null`,
+  "not-null": t`not null`,
+};
+
+const STRING_OPERATOR_NAMES = {
+  "=": t`equal to`,
+  "!=": t`not equal to`,
+  "is-null": t`null`,
+  "not-null": t`not null`,
```

Comment:
For consistency with how Metabase filters work I think you should also add contains, does not contain, starts with and ends with

Reply:
Done